src/evaluation.py

```diff
@@ -352,10 +354,41 @@ def match_answer_for_question(self, current_score, question, marked_answer):
         )
         return delta
 
-    def conditionally_print_explanation(self):
+    def conditionally_print_explanation(self, file_id):
         if self.should_explain_scoring:
             console.print(self.explanation_table, justify="center")
 
+        self.explanation_to_csv(file_id)
+        self.explanation_table_data_for_csv = []
+
+    # Explanation Table to CSV
+    def explanation_to_csv(self, file_id):
+        evaluation_json = open_evaluation_with_validation(self.path)
+
+        if evaluation_json["options"].get("enable_evaluation_table_to_csv", False):
```

Comment:
We should already have access to options in constructors. No need to load the json file again.
```
self.should_explain_scoring = options.get("should_explain_scoring", False)
self.enable_evaluation_table_to_csv = options.get("enable_evaluation_table_to_csv", False)
```